src/library/pia/pia-format.ts

```diff
@@ -1014,13 +1025,16 @@ export class PiaFormat {
     this.piaData.oasdiEarnings = mapMap(oasdiEarnings, (v, k) =>
       (v && v === -1) || v === "-1" ? 0 : v
     );
-
-    this.piaData.firstEarningYearActual = Math.min(
-      ...Array.from(oasdiEarnings.keys())
-    );
-    this.piaData.lastEarningYearActual = Math.max(
-      ...Array.from(oasdiEarnings.keys())
-    );
+    //TODO - this may not necessary map to first and last year of reported earnings.
+    // this includes both projected and entered earnings.
+    //We may need to remove this line altogether.
+    //this.piaData.firstEarningYear = Math.min(
+    //  ...Array.from(oasdiEarnings.keys())
+    //);
```

Comment:
I suspect taking this out of our library breaks our current website. Should we move this code to the front end and call the setters for the first and last years by hand there?